### ISSUE TITLE
tfsec v1.28.2 breaks the action

### ISSUE THREAD

**Issue by kyontan** [suzuki-shunsuke/github-action-tfsec#618]
(the issue is originally created by @44smkn)

tfsec v1.28.2 added a transition message. 
https://github.com/aquasecurity/tfsec/pull/2067

tfsec prints a message even if the JSON format is specified.  

This action (`suzuki-shunsuke/github-action-tfsec`) tries to parse stdout from tfsec.
But, if fails with the message.

```
======================================================
tfsec is joining the Trivy family

tfsec will continue to remain available 
for the time being, although our engineering 
attention will be directed at Trivy going forward.

You can read more here: 
https://github.com/aquasecurity/tfsec/discussions/1994
======================================================
{
	"results": []
}
Parsing tfsec result
Error: Unexpected token = in JSON at position 1
```

`scaffold working directory` also has the same issue.
It installs the latest version tfsec. It is v1.28.2 now.
https://github.com/suzuki-shunsuke/tfaction/blob/main/scaffold-working-dir/action.yaml#L52-L55

**Comment by suzuki-shunsuke** [comment]
Thank you for reporting!

It seems there is a pull request resolving the issue, so maybe this issue will be solved by tfsec.
- https://github.com/aquasecurity/tfsec/pull/2099

But we should also consider handling this issue ourselves.

**Comment by suzuki-shunsuke** [comment]
https://github.com/suzuki-shunsuke/github-action-tfsec/blob/19ec9a787d536c6e89fb656a91a90749fca13137/src/run.ts#L90-L95

**Comment by suzuki-shunsuke** [comment]
- https://github.com/suzuki-shunsuke/github-action-tfsec/pull/620

**Comment by suzuki-shunsuke** [comment]
This issue was resolved by #620 .
And v0.1.9 is out 🎉
https://github.com/suzuki-shunsuke/github-action-tfsec/releases/tag/v0.1.9
